src/routes/GettingStartedPage/Wizard/InstallOptions.js

```diff
@@ -69,7 +69,7 @@ const InstallOptions = ({
       <StackItem>
         <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
           <FlexItem>
-            <Title headingLevel="h1">Installation Options</Title>
+            <Title headingLevel="h1">SecuredCluster Installation Options</Title>
```

Comment:
Should this be `Secured Cluster` or is it actually `SecuredCluster`?

Reply:
I'm pretty sure it's supposed to be one word but I asked the same question. I'll wait on a response before I merge.